WISE-Community/WISE-Client
# feat(Peer Chat): Implement moderation

Test with
- https://github.com/WISE-Community/WISE-API/pull/210

## Changes

Allow teachers to delete and undelete Peer Chat messages.

## Test

- Make sure Peer Chat messages the teacher has deleted do not show up for the student
- Make sure Peer Chat messages the teacher has deleted and then undeleted do show up for the student
- Make sure teachers can delete student and teacher Peer Chat messages

Closes #1027
